lunes_cms/cms/fixtures/test_data.json

```diff
@@ -14711,7 +14711,7 @@
     "pk": 253,
     "fields": {
       "alt_word": "Papierkorb",
-      "article": 1,
+      "singular_article": 1,
       "document": 142
```

Comment:
```suggestion
      "document": 142,
      "plural": "Papierkörbe",
      "plural_article": 1,
```
I think it could make sense to add some sample data. I think adding a test would make sense too, but this could also be done in a second PR.